muacrypt/recommendation.py

```diff
@@ -6,8 +6,34 @@ def __init__(self, account, peerstates):
         self.peerstates = peerstates
 
     def ui_recommendation(self):
-        return 'available'
+        # only consider first peer for now
+        peer = list(self.peerstates.values())[0]
+        return self._peer_recommendation(peer)
 
     def target_keys(self):
-        return {addr: state.public_keyhandle for addr, state in
+        return {addr: self._target_key(peer) for addr, peer in
                 self.peerstates.items()}
+
+    def _peer_recommendation(self, peer):
+        if peer is None:
+            return 'disable'
```

Comment:
looking at the code paths i doubt that peer can ever be none. ``states.get_peerstate(account_name, addr)`` always returns a PeerState object, never None.

Reply:
Yes. I mainly added this because it's in the spec. Then I though... will never be needed... then i thought... well i can be closer to the spec and more flexible in what i accept.

Should i assert peer instead?

Reply:
the spec says "If there is no peers[to-addr], then set ui-recommendation to disable, and terminate. For the purposes of the rest of this recommendation, if either public_key or gossip_key is revoked, expired, or otherwise known to be unusable for encryption, then treat that key as though it were null (not present)." -- so we could translate that as ``has_usable_key`` or so (and if we don't have one, we are at ``disable``).  No need to ``assert peer is not None`` -- python will already blow up with an exception if ``has_usable_key`` is called on None.